.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,54 +6,40 @@ labels: "bug"
 assignees: ""
 ---
 
-**Search terms you've used**
+<!-- 
+Thank you for reporting an issue.
 
-<!-- What search terms have you used to check whether this bug was already reported? -->
-
-**Describe the bug**
-
-<!-- A clear and concise description of what the bug is. -->
-
-**To Reproduce**
-Steps to reproduce the behavior:
-
-1. Given this data: `…`
-2. Call this function `…`
-3. See error
-
-**CodeSandbox**
-
-<!--
-Adding a minimal reproduction of the bug makes it as easy as possible to get it fixed.
-To do so, fork the following CodeSandbox, and share the resulting link here:
-
-https://codesandbox.io/s/github/inrupt/solid-client-js/tree/master/.codesandbox/sandbox
+Please fill in as much of the template below as you're able. Feel free to delete any section you want to skip. 
```

Comment:
As in the other PR, this is good, but I'd still encourage people to fill in as much as they can.

```suggestion
Please fill in as much of the template below as you're able. Feel free to delete any section you don't think applies, but be aware that the more comprehensive your description, the easier it is to take it into account. 
```